src/textual/widgets/text_input.py

```diff
@@ -123,6 +136,17 @@ def __init__(
         self.placeholder = placeholder
         self._editor = TextEditorBackend(initial, 0)
         self.visible_range: tuple[int, int] | None = None
+        self.autocompleter = autocompleter
+        self._suggestion_suffix = ""
+
+        self._cursor_blink_visible = True
+        self._cursor_blink_timer: Timer | None = None
+        self._last_keypress_time: float = 0.0
+        if self.cursor_blink_enabled:
+            self._last_keypress_time = time.monotonic()
```

Comment:
Now that we have [this PR](https://github.com/Textualize/textual/pull/507) merged we should avoid direct calls to `time.monotonic()`, and use the new `textual._clock` module as our internal single source of truth for everything related to time :slightly_smiling_face: 
So here it would be `self._last_keypress_time = _clock.get_time_no_wait()` 

(`_clock` exposes 3 functions: `get_time_no_wait() -> float`, `async get_time() -> float` (same, but asynchronous - useful in some cases) and `async sleep(seconds: float) -> None`

Reply:
Would you mind making that change @darrenburns